### PR TITLE
License meta data

### PR DESCRIPTION
**Pull request by oskarkvamme**
- ArticleByLine: Fix correct license info. BREAKING CHANGES: license prop is now a object (use getLicenseByAbbreviation).
- Figure: Support for title and dynamic license text and link. BREAKING CHANGES: messages.learnAboutOpenLicenses -> learnAboutLicenses. New props: title, licenseUrl(use getLicenseByAbbreviation)
- LicenseIconDescriptionList and LicenseIconList: Remove hard coded CC license. CC is now returned from getLicenseRightByAbbreviation
- Design manual 'Sammensatte lisenser': List of licenses with details
- ndla-licenses licenses.js: added url and link text to license definition. Added licenses: pd, cc0 and copy
- ndla-licenses microdata.js help functions for rendring of microdata
- MediaList: Implemented micro data and cc rel meta data.
- MediaListCCLink:  BREAKING CHANGES: new prop url. Link to license (use getLicenseByAbbreviation)
- MediaListItemBody: BREAKING CHANGES: new props  resourceUrl and resourceType. resourceUrl is the url to the media (video, image, h5p etc.). It resourceUrl is not defined it is defaulted to current url. resourceType is which type the resource is (video, image etc.) 
 - MediaListItemMeta: BREAKING CHANGES: changes in items prop. New property metaType(says what type of item it is (author, copyrightHolder, tiltle etc.). New property microDataType: specifies if the item is a person or a organization (is not needed to specify). Defaults to person on author, organization on copyrightHolder and person on contributer). For other types it is not in use (title, other etc.).

### CODE REVIEW

File: packages/designmanual/stories/pages/ArticleExercise.jsx [hunk 1]
```diff
@@ -63,7 +65,9 @@ export default () => (
           tydeligere for både deg selv og dem du eventuelt jobber sammen med i
           klassen.
         </p>
-        <FigureWithLicense caption="">
+        <FigureWithLicense
+          caption=""
+          resourceUrl="https://staging.api.ndla.no/image-api/raw/42-45210905.jpg?width=1000">
```
Comment (oyvinmar) [inline packages/designmanual/stories/pages/ArticleExercise.jsx]: Ser ikke at resourceUrl blir brukt i FigureWithLicense. Noe som burde fjernes eller fikses i FigureWithLicense?

File: packages/designmanual/stories/article/LicenseExample.jsx [hunk 2]
```diff
@@ -40,20 +55,36 @@ const VideoContent = () => (
...
         </MediaListItemImage>
         <MediaListItemBody
-          license="by-nc-nd"
-          title="Regler for bruk av videoen:">
-          <MediaListCCLink>Lær mer om åpne lisenser</MediaListCCLink>
+          license={byncndLicenseAbbreviation}
+          title="Regler for bruk av videoen:"
+          resourceUrl="https://www.youtube.com/embed/f9VriNNRn0U?feature=oembed"
+          resourceType="video">
+          <MediaListCCLink url={byncndLicense.url}>
```
Comment (oyvinmar) [inline packages/designmanual/stories/article/LicenseExample.jsx]: Siden lisensen sendes inn til MediaListItemBody og teksten til linken nå ligger i licenses, så foreslår jeg å bruke <MediaListCCLink> direkte i MediaListItemBody komponenten. Altså ikke send den inn via children.

File: packages/designmanual/stories/pages/ArticleExercise.jsx (hunk 1 above)
Comment (oyvinmar) [reply packages/designmanual/stories/pages/ArticleExercise.jsx]: Eller kommer det i en annen PR?

Comment (oskarkvamme) [reply packages/designmanual/stories/pages/ArticleExercise.jsx]: det er gammel kode, så kan fjernes

File: packages/designmanual/stories/article/LicenseExample.jsx (hunk 2 above)
Comment (oskarkvamme) [reply packages/designmanual/stories/article/LicenseExample.jsx]: Enig

**Pull request merged**